modules/gateway/controllers/service/populate/virtualservice.go

```diff
@@ -110,7 +110,7 @@ func newRoute(systemNamespace, externalGW string, published bool, portBinding v1
 			ns = svc.Namespace
 		}
 		host := dests[0].Host
-		if dests[0].Subset != "" {
+		if dests[0].Subset != "" && dests[0].Subset != "cm-acme" {
```

Comment:
Should probably be a constant if we are using in multiple places

Reply:
:+1: